### ISSUE TITLE
Syn and Real results?

### ISSUE THREAD

**Issue by wangkaihong** [JitengMu/Learning-from-Synthetic-Animals#7]
Hi,

Thanks for sharing this excellent work! Just wanted to check how exactly did you get the results of "source only" in Tab. 1? When I trained the model with only synthetic data as indicated in Example 2 in TRAINING.md, the result is ~60% for "horse" and ~50% for "tiger", rather than 25.33% and 21.17% in the paper. 

You may also find similar things happen in: 
https://github.com/chaneyddtt/UDA-Animal-Pose/blob/main/README.md

where the following work that heavily borrowed your code stated: "The pretrained model achieve a PCK@0.05 of 64.71 on the real animal dataset" when they also pretrained the model on only synthetic data. Would you clarify how you get your results?

Additionally, for your "synthetic + real" results in Tab. 1, did you train the model on both the labeled source and labeled target domain dataset but not only on the labeled target domain dataset? Was the loss function for the training on the target domain data the same as that on source data? Would it be possible to share your training script for that experiment?

Thanks in advance for your time.

**Comment by JitengMu** [comment]
Thanks for your interest. 

a ) I think the main difference is whether strong augmentation is applied when training on synthetic data. As stated in the paper Section 4.2, the Table 1 syn only baseline is trained without strong augmentation. When trained with strong data augmentation, the results are shown as Table 3 Baseline, which roughly matches your experiment results. I am not sure if the follow-up paper uses exactly the same settings, maybe contacting the corresponding authors would better answer the question. 

b ) Sorry the supervised joint training script was not released. However, it should be easy to modify the CCSSL script to obtain the results. The experiment was conducted similar to CCSSL: a model trained on synthetic data with strong augmentation is obtained first, and then a joint supervised training on both synthetic and real data will yield the final results. Note the only difference is that now the real data labels are given so you don't have to use pseudo-labels.

Hope this addresses your concerns. Let me know if you have any further questions.

**Comment by wangkaihong** [comment]
That makes sense to me, thanks for your clarification!
